contractcourt/chain_watcher.go

```diff
@@ -590,6 +613,15 @@ func (c *chainWatcher) dispatchLocalForceClose(
 		closeSummary.TimeLockedBalance += htlcValue
 	}
 
+	// Attempt to add a channel sync message to the close summary.
```

Comment:
What about remote force close?

Reply:
Good catch! Fixed.

Reply:
Don't see it fixed in the latest diff: https://github.com/lightningnetwork/lnd/blob/4124c6db8159bf79327181029f613fc9d404f4a1/contractcourt/chain_watcher.go#L654